test/conf_test.py

```diff
@@ -17,16 +17,11 @@
 sys.argv.extend([
     '--config-value=flag_key=flag_value',
     '--config-value', 'other_flag=other_value',
-# You can specify arbitrary keys, but they'll get ignored if they aren't
```

Comment:
ah good call, i always forget that all the tests get run in the same Python instance.